api/api/serializers/docs.py

```diff
@@ -0,0 +1,84 @@
+from django.conf import settings
+
+from api.constants.parameters import TAG
+
+
+CREATOR_COLLECTIONS_DISABLED = """
+Search by creator only. Cannot be used with `q`. The search
+is fuzzy, so `creator=john` will match any value that includes the
+word `john`. If the value contains space, items that contain any of
+the words in the value will match. To search for several values,
+join them with a comma."""
+
+CREATOR = """
+_When `q` parameter is present, `creator` parameter is ignored._
+
+**Creator collection**
+When used with `collection=creator`, returns the collection of media
```

Comment:
Since the source is also required here, can we make that explicit in the example:
```suggestion
When used with `collection=creator&source=sourceName`, returns the collection of media
```